Makefile

```diff
@@ -43,6 +43,8 @@ upgrade: $(COMMON_CONSTRAINTS_TXT)  ## update the requirements/*.txt files with
 	pip-compile --upgrade -o requirements/pip_tools.txt requirements/pip_tools.in
 	pip install -qr requirements/pip.txt
 	pip install -qr requirements/pip_tools.txt
+	sed 's/Django<4.0//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
```

Comment:
This constraint override is not needed because global constraint has already been updated.

Reply:
@awais786 can you make this change then I think we can merge and release this.